spec/lib/miq_expression_spec.rb

```diff
@@ -2423,6 +2423,21 @@
       expect(exp.to_human).to eq("COUNT OF Snaps > 1")
     end
 
+    it "generates a human readable string for a numeric FIELD value expression" do
+      exp = MiqExpression.new(">" => {"field" => "Vm.hardware.volumes-free_space_percent", "value" => 75})
+      expect(exp.to_human).to eq("VM and Instance.Hardware.Volumes : Free Space Percent > 75")
+    end
+
+    it "generates a human readable string for a numeric FIELD value expression" do
```

Comment:
This text is the same as the previous test, so I think change to 
```suggestion
    it "generates a human readable string for a float FIELD value expression" do
```